src/invenio-records-js/controllers/InvenioRecordsCtrl.js

```diff
@@ -155,78 +155,83 @@ function InvenioRecordsCtrl($scope, $rootScope, $q, $window, $location,
   }
 
   /**
-    * Records actions
+    * Do a data massage before sending with request
     * @memberof InvenioRecordsCtrl
-    * @function invenioRecordsActions
-    * @param {Object} evt - The event object.
-    * @param {String} type - The invenio action type.
-    * @param {String} method - The invenio request method.
-    * @param {Object} successCallback - Call function after success.
-    * @param {Object} errorCallback - Call function after error..
+    * @function cleanData
     */
-  function invenioRecordsActions(evt, type, method, successCallback, errorCallback) {
+  function cleanData() {
+    var _data = angular.merge({}, {metadata: vm.invenioRecordsModel});
```

Comment:
It's already wrapped in the current version https://github.com/inveniosoftware/invenio-records-js/blob/master/src/invenio-records-js/controllers/InvenioRecordsCtrl.js#L187-L189  It will change in https://github.com/inveniosoftware/invenio-records-js/pull/24